core/db/migrate/20221130103509_remove_taxon_position.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTaxonPosition < ActiveRecord::Migration[ActiveRecord::Migration.current_version]
```

Comment:
To ensure syntax compatibility, we're creating migrations always with the oldest version in mind:

https://github.com/solidusio/solidus/blob/efe2b3a20d0577413e4810c008ea564e637c36a3/core/db/migrate/20220805202442_add_level_to_spree_tax_rates.rb#L1